packages/react-components/src/components/Survey/StarSurvey/StarSurvey.styles.ts

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IModalStyles, IRatingStyles, Theme, mergeStyles } from '@fluentui/react';
+import { _pxToRem } from '@internal/acs-ui-common';
+
+/**
+ * @private
+ */
+export const questionTextStyle = (theme: Theme): string =>
+  mergeStyles({
+    fontWeight: 600,
+    fontSize: _pxToRem(20),
+    lineHeight: _pxToRem(28),
+    color: theme.palette.neutralPrimary,
+    marginRight: _pxToRem(24)
+  });
+
+/**
+ * @private
+ */
+export const helperTextStyle = (theme: Theme): string =>
+  mergeStyles({
+    fontWeight: 400,
+    fontSize: _pxToRem(12),
+    lineHeight: _pxToRem(16),
+    color: theme.palette.neutralSecondary,
+    paddingTop: _pxToRem(20)
+  });
+
+/**
+ * @private
+ */
+export const ratingStyles = (): Partial<IRatingStyles> => {
+  return {
+    root: {
+      marginBottom: _pxToRem(24),
+      textAlign: 'center'
+    },
+    ratingStar: {
```

Comment:
Is there any styles we can add here to center the focus box? from the splash image in the PR the focus box is not centered around the icon